docs/examples.rst

```diff
@@ -0,0 +1,388 @@
+Invocation Examples
+===================
+
+These are examples of how to invoke the bdb command, auto-generated from the tests.
+
+
+create
+------
+
+
+.. code-block:: shell
+
+   $ CONDITIONS='{
+       "amount": 1,
+       "condition": {
+           "details": {
+               "bitmask": 32,
+               "public_key": "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15",
+               "signature": null,
+               "type": "fulfillment",
+               "type_id": 4
+           },
+           "uri": "cc:4:20:zL3F_XLRs_snrfmdqSFPqEcu-bu1xF6636oSYpNWvIw:96"
+       },
+       "owners_after": [
+           "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15"
+       ]
+   }'
+
+
+   $ bdb create 35qDXhZTUvna23NLc1hMfmrgPniBwPgNjko1VfQuD3vF '$CONDITIONS'
```

Comment:
Not if you're passing a JSON object, I would have thought. This is shell script remember.